src/table/src/table/scan.rs

```diff
@@ -73,12 +73,19 @@ impl RegionScanExec {
         }
 
         let metadata = scanner.metadata();
-        let mut pk_columns: Vec<PhysicalSortExpr> = metadata
+        let mut pk_names = metadata
             .primary_key_columns()
+            .map(|col| col.column_schema.name.clone())
+            .collect::<Vec<_>>();
+        // workaround for logical table
```

Comment:
s this because the primary key (pk) order in the logical table is inconsistent with the primary key order in the physical table, and the primary key orders between logical tables are also inconsistent, so a sort operation was performed here?

Reply:
yes, metric engine's physical table and logical table have different pk order

Reply:
Okay. Leaving a FIXME here, as suggested above.